src/block/video-popup/save.js

```diff
@@ -54,6 +54,11 @@ export const Save = props => {
 			attributes={ attributes }
 			data-video={ attributes.videoLink }
 			version={ props.version }
+			{ ...{
+				'data-nofullscreen': attributes.videoFullscreen ? null : '',
+				'data-nodownload': attributes.videoDownload ? null : '',
+				'data-loop': attributes.videoLoop ? '' : null,
+			} }
```

Comment:
You can place the props directly instead of adding them to an object and spreading it
```suggestion
			data-nofullscreen={ attributes.videoFullscreen ? null : '' }
			data-nodownload={ attributes.videoDownload ? null : '' }
			data-loop={ attributes.videoLoop ? '' : null }
```